packages/clerk-js/src/ui/utils/test/fixtureHelpers.ts

```diff
@@ -36,10 +37,42 @@ export const createClientFixtureHelpers = (baseClient: ClientJSON) => {
 };
 
 const createUserFixtureHelpers = (baseClient: ClientJSON) => {
-  type WithUserParams = Omit<Partial<UserJSON>, 'email_addresses' | 'phone_numbers' | 'external_accounts'> & {
+  type OrgParams = Partial<OrganizationJSON> & { role?: MembershipRole };
+
+  type WithUserParams = Omit<
+    Partial<UserJSON>,
+    'email_addresses' | 'phone_numbers' | 'external_accounts' | 'organization_memberships'
+  > & {
     email_addresses?: Array<string | Partial<EmailAddressJSON>>;
     phone_numbers?: Array<string | Partial<PhoneNumberJSON>>;
     external_accounts?: Array<OAuthProvider | Partial<ExternalAccountJSON>>;
+    organization_memberships?: Array<string | OrgParams>;
+  };
+
+  const createOrganization = (params: OrgParams): OrganizationMembershipJSON => {
+    const { role, ...orgParams } = params;
+    return {
+      created_at: new Date().getTime(),
+      id: orgParams?.id || orgParams?.name || 'test_id',
+      object: 'organization_membership',
+      organization: {
+        created_at: new Date().getTime(),
+        id: orgParams?.id || orgParams?.name || 'test_id',
```

Comment:
No need for `orgParams?.id` here as it'll be overridden if provided!

Reply:
@desiprisg Well, if you provide only `id` it will not be overwritten I guess.

Reply:
Wouldn't line 70 overwrite it?

Reply:
Oops, yes, you are right!